history-api/script.js

```diff
@@ -0,0 +1,43 @@
+// Update the page with the new content
+function displayContent(content) {
+  const description = document.querySelector("#description");
+  description.textContent = content.description;
+
+  const photo = document.querySelector("#photo");
+  photo.setAttribute("src", content.image.src);
+  photo.setAttribute("alt", content.image.alt);
+}
+
+// Handle click on link
```

Comment:
NIT
```suggestion
// When a link is clicked: update the page for that creature.
// From a user perspective, this will look like the loading of a new page.

```